fdbclient/NativeAPI.actor.cpp

```diff
@@ -2140,6 +2140,12 @@ Future< Void > Transaction::watch( Reference<Watch> watch ) {
 ACTOR Future< Standalone< VectorRef< const char*>>> getAddressesForKeyActor( Key key, Future<Version> ver, Database cx, TransactionInfo info ) {
 	state vector<StorageServerInterface> ssi;
 
+	state bool includePort = false;
```

Comment:
I suppose based on the comment below that this should be API versioned. I guess that's a little weird considering that we already have a 6.2 version supporting API version 620 that doesn't do this, but it seems like the right thing to do. The most likely case where I could imagine this actually mattering is somebody running this function on `\xff\xff` because it happens to be a shard boundary key, and while before that would return an empty list, now it will return a list of data with different format.

Reply:
I guess in theory, we could use something like API version 621 for this. I'm not sure what the general opinion is about doing that.

Reply:
An alternative that sidesteps the versioning issue is to use an option or even a knob.

Reply:
I actually didn't think about the option - it might actually be cleaner than the `\xff\xff`-hack. @atn34 what are your thoughts on this? We could add the same option to FDB3 (as long as you can set in on the database level) and then set this on the client side